src/main/java/org/opentripplanner/apis/gtfs/support/filter/PatternByDateFilterUtil.java

```diff
@@ -0,0 +1,23 @@
+package org.opentripplanner.apis.gtfs.support.filter;
+
+import org.opentripplanner.apis.gtfs.generated.GraphQLTypes;
+import org.opentripplanner.apis.gtfs.model.LocalDateRange;
+import org.opentripplanner.transit.service.PatternByServiceDatesFilter;
+import org.opentripplanner.transit.service.TransitService;
+
+/**
+ * Utility methods for instantiating a {@link PatternByServiceDatesFilter}/
```

Comment:
```suggestion
 * Utility methods for instantiating a {@link PatternByServiceDatesFilter}.
```

Reply:
I think the `/` was a mistake. but if not, feel free to ignore this suggestion.

Reply:
It was. Applied.